### PR TITLE
[Doc improvement] [SBS Meeting Stage View][2352819]

### PR DESCRIPTION
**Pull request by v-abirade**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-meetings-stage-view.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/41f7cb6539855dcb441f981d470e16b2fdb45c9e/msteams-platform%2Fsbs-meetings-stage-view.yml) | 88 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/01c2ac01-65a0-4af8-9a04-9895c04622fa) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e5d95cb66a4d19bd3057a31879bd8c2128a04d09/msteams-platform%2Fapps-in-teams-meetings%2Fenable-and-configure-your-app-for-teams-meetings.md) | 97 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ed1330bb-41f9-499e-a0b1-26b16ff34664) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-meetings-stage-view.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/e5d95cb66a4d19bd3057a31879bd8c2128a04d09/msteams-platform%2Fsbs-meetings-stage-view.yml) | 88 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cfdee77a-dbb0-4fea-81b6-73cdb93449ee) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a66861b90998d5bd616e2cae512535890c43865e/msteams-platform%2Fapps-in-teams-meetings%2Fenable-and-configure-your-app-for-teams-meetings.md) | 97 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/29bbd708-578f-4c07-b635-5c2cdb10fea0) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/a66861b90998d5bd616e2cae512535890c43865e/msteams-platform%2Fmstdd-landing.yml) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2734cd35-0322-4866-af93-eb2a1a13166e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-meetings-stage-view.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/a66861b90998d5bd616e2cae512535890c43865e/msteams-platform%2Fsbs-meetings-stage-view.yml) | 88 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f815dd53-c566-4bc2-8787-655655c59d05) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9baeaa799496a35dbc5417b3bf95a2a673a6c8ba/msteams-platform%2Fapps-in-teams-meetings%2Fenable-and-configure-your-app-for-teams-meetings.md) | 97 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2d12ce98-caf8-4c9a-a398-9e6da240b65a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/9baeaa799496a35dbc5417b3bf95a2a673a6c8ba/msteams-platform%2Fmstdd-landing.yml) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b0be0c33-8129-4dd7-9bf7-08bc8dd2a4d7) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-meetings-stage-view.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/9baeaa799496a35dbc5417b3bf95a2a673a6c8ba/msteams-platform%2Fsbs-meetings-stage-view.yml) | 88 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e80f11e9-daa5-4a57-aa34-59945215b153) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3aceda718bf16632e54b67be0e257b50ff12a6c3/msteams-platform%2Fapps-in-teams-meetings%2Fenable-and-configure-your-app-for-teams-meetings.md) | 97 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9f7f0994-7e02-4616-9828-cc1a15ed8803) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/3aceda718bf16632e54b67be0e257b50ff12a6c3/msteams-platform%2Fmstdd-landing.yml) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b59ef9e6-ad41-44b9-a450-17be3cac1f25) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-meetings-stage-view.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/3aceda718bf16632e54b67be0e257b50ff12a6c3/msteams-platform%2Fsbs-meetings-stage-view.yml) | 88 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c845e65a-86a0-4dbd-85de-e28bd0d9f56b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e6e5aff6381d48360fd4ea38c4abcf2ecde0a2f6/msteams-platform%2Fapps-in-teams-meetings%2Fenable-and-configure-your-app-for-teams-meetings.md) |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [msteams-platform/sbs-meetings-stage-view.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/e6e5aff6381d48360fd4ea38c4abcf2ecde0a2f6/msteams-platform%2Fsbs-meetings-stage-view.yml) |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [msteams-platform/sbs-teams-connectors.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/e6e5aff6381d48360fd4ea38c4abcf2ecde0a2f6/msteams-platform%2Fsbs-teams-connectors.yml) |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/sbs-meetings-stage-view.yml [hunk 1]
```diff
@@ -14,102 +14,108 @@ metadata:
...
       > After downloading ngrok, sign up and install [authtoken](https://ngrok.com/download).
 
-    Ensure you have working knowledge of the following technologies:
+     You must have working knowledge of the following technologies:
 
     * [C#](https://docs.microsoft.com/dotnet/csharp/fundamentals/tutorials/how-to-display-command-line-arguments)
     * [ECMAScript6](http://es6-features.org/#Constants)
-    * [ASP.NET core](https://docs.microsoft.com/aspnet/core/?view=aspnetcore-3.1) version 3.1
-    * [React.JS](https://reactjs.org/tutorial/tutorial.html) version 16 or later
+    * [Latest version of ASP.NET core](https://docs.microsoft.com/aspnet/core/?view=aspnetcore-3.1)
```
Comment (eoinobrien) [inline msteams-platform/sbs-meetings-stage-view.yml:46]: 3.1 is not the latest version of Dotnet. Also, the latest version of dotnet does not have the 'core' suffix

Comment (v-tmukherjee) [reply msteams-platform/sbs-meetings-stage-view.yml:46]: Resolved

**Review by v-abirade — COMMENTED** [review]
Approved

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9cecf7bcc4467abd3c17afabc94a9c7c876b5e01/msteams-platform%2Fapps-in-teams-meetings%2Fenable-and-configure-your-app-for-teams-meetings.md) | 97 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a35d2fc5-b767-496d-8370-149ca21e7539) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-meetings-stage-view.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/9cecf7bcc4467abd3c17afabc94a9c7c876b5e01/msteams-platform%2Fsbs-meetings-stage-view.yml) | 88 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4d5f0ba6-170e-4248-b98a-fe05e6808959) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-teams-connectors.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/9cecf7bcc4467abd3c17afabc94a9c7c876b5e01/msteams-platform%2Fsbs-teams-connectors.yml) | 82 | 36 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7ff091fa-faec-44d7-ab93-58ca89fb95a2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**